### PR TITLE
Importer resources updates.

### PR DESCRIPTION
**Pull request by ghost**
- new resource: DiscoveryDataCenterResource
- network resource imports network terminators
- fixed result of import msg

### CODE REVIEW

File: src/ralph/data_importer/resources.py [hunk 1]
```diff
@@ -182,6 +182,25 @@ class Meta:
         model = physical.Rack
 
 
+class DiscoveryDataCenterResource(RalphModelResource):
+    class Meta:
+        model = physical.DataCenter
+
+class NetworkKindResource(RalphModelResource):
```
Comment (ghost) [inline src/ralph/data_importer/resources.py]: perhaps, useless

File: src/ralph/data_importer/resources.py [hunk 2]
```diff
@@ -182,6 +183,11 @@ class Meta:
         model = physical.Rack
 
 
+class DiscoveryDataCenterResource(RalphModelResource):
```
Comment (mkurek) [inline src/ralph/data_importer/resources.py:186]: please add some comment that "this is another resource for `DataCenter` to handle another source of data from Ralph2 (discovery DataCenter)"

File: src/ralph/data_importer/management/commands/importer.py [hunk 3]
```diff
@@ -166,12 +166,19 @@ def from_file(self, options):
...
-            else:
-                self.stdout.write(
-                    '{} rows were imported'.format(len(csv_body))
+
+            self.stderr.write(
+                'Imported records: {}'.format(
+                    after_import_count - before_import
+                )
+            )
+            if (len(csv_body) - after_import_count - before_import >= 0)
```
Comment (mkurek) [inline src/ralph/data_importer/management/commands/importer.py]: missing colon

File: src/ralph/data_importer/resources.py (hunk 2 above)
Comment (ghost) [inline src/ralph/data_importer/resources.py:186]: done

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**